spidermon/contrib/scrapy/monitors/base.py

```diff
@@ -1,18 +1,86 @@
+import operator
+import logging
+
 from spidermon import Monitor
 from spidermon.exceptions import NotConfigured
 
 from ...monitors.mixins.spider import SpiderMonitorMixin
 
+logger = logging.getLogger(__name__)
+
 
 class BaseScrapyMonitor(Monitor, SpiderMonitorMixin):
     longMessage = False
+    ops = {"==": operator.eq, "<": operator.lt, ">": operator.gt}
```

Comment:
Maybe we should support the same operations as https://github.com/scrapinghub/spidermon/blob/e08588691f31e13f2e10233f2711632ad2d49294/spidermon/contrib/scrapy/monitors/base.py#L194-L201, and maybe we could add a test that verifies that they keywords in these 2 dicts remain the same (i.e. so that, if we add an operation to one dict, we are reminded to also add it to the other).